### DISCUSSION TITLE
Import Globally On Demand

### DISCUSSION THREAD

**Discussion by adityafawzan** [tusen-ai/naive-ui#2542] [Q&A]
how to import this in App.vue:

import { darkTheme } from 'naive-ui'
import type { GlobalTheme } from 'naive-ui'

just like these components : 
import {
  // create naive ui
  create,
  // component
  NButton,
  NLayout,
  NLayoutHeader,
  NLayoutSider,
  NLayoutFooter,
  NH2,
} from "naive-ui";

const naive = create({
  components: [NButton, NLayout, NLayoutHeader, NLayoutSider, NLayoutFooter, NH2],
});

**Comment by 07akioni** [comment]
I can't understand what's the problem. Import it just like other libraries you are using, no difference.

**Reply by adityafawzan** [reply]
i meant, how to import it globally in App.vue, so i can use it all across the templates just like the components ?

**Reply by 07akioni** [reply]
```
// your code, then

const app = createApp()

app.use(naive)
```
